src/core.js

```diff
@@ -1680,11 +1680,17 @@ define([
                 isCopyable: !multiselect && mug.options.isCopyable
             }));
         $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true);
-            if (mugs.length > 1 || (mugs.length && form.getChildren(mugs[0]).length)) {
+            var mugs = _this.getCurrentlySelectedMug(true),
+                errorMessage;
+            if (mugs.length > 1) {
+                errorMessage = "You are about to delete multiple questions. This cannot be undone";
+            } else if (mugs.length && form.getChildren(mugs[0]).length) {
+                errorMessage = "You are about to delete a question with child nodes. This cannot be undone";
```

Comment:
Ah, so she was confused at seeing it sometimes but not others, which is fair. I think adding a few extra words about deleting multiple questions should give a hint about why it shows up sometimes but not others.

I love how she concludes:
> Also, though good for new app builders, this pop up box is actually pretty annoying if you’re deleting questions en masse and know what you’re doing.

Right on. I think adding too much more text could make it even more annoying. :)